source/includes/limitations-rs-to-sharded.rst

```diff
@@ -0,0 +1,12 @@
+- ``mongosync`` allows users to rename collections that the
+  ``sharding.shardingEntries`` option includes during sync. To see limitations 
+  on renaming collections while ``mongosync`` is running, see :ref:`Renaming
+  During Sync <rename-during-sync>`.
```

Comment:
[suggestion] for minimalism

```suggestion
  ``sharding.shardingEntries`` option includes during sync. To see 
  renaming limitations, see :ref:`Renaming
  During Sync <rename-during-sync>`.
```

Reply:
Thanks for the suggestion, changing now!